src/pages/TransactionDuplicate/ReviewBillable.tsx

```diff
@@ -23,22 +23,22 @@ function ReviewBillable() {
         () =>
             compareResult.change.billable?.map((billable) => ({
                 text: billable ? translate('common.yes') : translate('common.no'),
-                value: billable,
+                value: billable ?? false,
             })),
         [compareResult.change.billable, translate],
     );
 
-    const onSelectRow = (data: FieldItemType) => {
+    const onSelectRow = (data: FieldItemType<'billable'>) => {
         if (data.value !== undefined) {
-            setReviewDuplicatesKey({billable: data.value as boolean});
+            setReviewDuplicatesKey({billable: data.value});
         }
         navigateToNextScreen();
     };
 
     return (
         <ScreenWrapper testID={ReviewBillable.displayName}>
             <HeaderWithBackButton title={translate('iou.reviewDuplicates')} />
-            <ReviewFields
+            <ReviewFields<'billable'>
```

Comment:
Nothing to address, today I just learned that you can do this for the name of the component 😄